degirum_tools/line_count.py

```diff
@@ -52,6 +52,37 @@ def __init__(self):
         self.for_class: Dict[str, SingleLineCounts] = {}
 
 
+class SingleVectorCounts:
+    """Class to hold vector crossing counts"""
+
+    def __init__(self):
+        self.in_direction: int = 0

```

Comment:
Not clear, what is in and what is out. Need to clarify in comments. Here, and in analyzer class.